speedy-recovery-client/src/components/secured/patientinformation/Practitioner/PractitionerInfo.js

```diff
@@ -1,39 +1,73 @@
 /*
-* Speedy Recovery -- A patient-centred app based on the FHIR standard facilitating communication between paediatric
-* patients, parents and hospital staff
-*
-* Copyright (C) 2019 University College London
-*
-* This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
-* Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option)
-* any later version.
-* This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
-* warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
-* details.
-* You should have received a copy of the GNU Affero General Public License along with this program. If not,
-* see http://www.gnu.org/license/.
-* */
+ * Speedy Recovery -- A patient-centred app based on the FHIR standard facilitating communication between paediatric
+ * patients, parents and hospital staff
+ *
+ * Copyright (C) 2019 University College London
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+ * Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+ * details.
+ * You should have received a copy of the GNU Affero General Public License along with this program. If not,
+ * see http://www.gnu.org/license/.
+ * */
 
 /* This file defines the PractitionerInfo component which creates an information page containing detailed information
 about each of the practitioners' patients.
  */
 
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { List, Image } from "semantic-ui-react";
 
 class PractitionerInfo extends Component {
-  render() {
-    const { patient } = this.props;
+  constructor(props) {
+    super(props);
+    this.state = {
+      patients: []
+    };
+  }
 
+  render() {
+    const patientItems = this.state.patients.map(patient => {
```

Comment:
instead of doing this.state.patients you can directly use this.props.patients

Reply:
If I directly use this.props.patients there will be a compile error, so I just keep this format

Reply:
whats the error you get? When i tried it I didnt see any errors